cvmfs/cvmfs.cc

```diff
@@ -338,6 +337,26 @@ static bool GetPathForInode(const fuse_ino_t ino, PathString *path) {
   return true;
 }
 
+static void DoTraceInode(const int event,
+                          fuse_ino_t ino,
+                          const std::string &msg)
+{
+  PathString path;
+  bool found = GetPathForInode(ino, &path);
+  if (!found) {
+    LogCvmfs(kLogCvmfs, kLogDebug,
+      "Tracing: Could not find path for inode %" PRIu64, uint64_t(ino));
+  } else {
+    mount_point_->tracer()->Trace(event, path, msg);
+  }
+}
+
+static void inline TraceInode(const int event,
```

Comment:
Because this is a common pattern, I'd add a `bool found_inode` to the parameter list and only trace if the inode is actually valid. This avoids the `if (found) ...` in all the places where TraceInode is called.